app/views/ReportView/components/RapidSummary/index.tsx

```diff
@@ -376,7 +386,7 @@ const RapidSummary = ({
         value: msiStatus,
       },
     ]);
-  }, [microbial, primaryBurden, tmburMutBur, report.m1m2Score, report.sampleInfo, report.tumourContent, report.captiv8Score, tCellCd8]);
+  }, [microbial, primaryBurden, tmburMutBur, report.m1m2Score, report.sampleInfo, report.tumourContent, tCellCd8.percentile, tCellCd8.score, report.captiv8Score, tCellCd8.percentileHidden, tCellCd8]);
```

Comment:
missing some dependencies? 

`tCellCd8.pedsScoreComment`  `tmburMutBur.adjustedTmb` `tmburMutBur.tmbHidden` `tCellCd8.pedsScore` `tCellCd8.pedsPercentile`

Reply:
oops fixing it now